src/main.ts

```diff
@@ -12,18 +12,19 @@ import * as coreclrdebug from './coreclr-debug/activate';
 import * as OmniSharp from './omnisharp/extension';
 import * as util from './common';
 import { Logger } from './logger';
-import { PackageManager, Status } from './packages';
+import { PackageManager, Status, PackageError } from './packages';
 import { PlatformInformation } from './platform';
 
 let _channel: vscode.OutputChannel = null;
+let _reporter: TelemetryReporter = null;
```

Comment:
Feels inconsistent to make this a global variable while we thread `logger` through. Should we just thread this through the functions with `logger`?